tests/test_config.toml

```diff
@@ -1,5 +1,5 @@
 [database]
-server="localhost"
+server="fidesctl-db"
```

Comment:
by leaving this as `localhost` I think what we were trying to do was make sure that a user could still run it locally without needing to be in a docker container. do you know what the side effects here are?

Reply:
This was the fix @SteveDMurphy and came up with when we were initially having test failures on the database. Once we made this change things started working again. With `localhost` the tests couldn't find the database.